Assets/WebGLTemplates/BoomTemplate/index.html

```diff
@@ -7,6 +7,7 @@
     <title>{{{ PRODUCT_NAME }}}</title>
     <link rel="shortcut icon" href="TemplateData/favicon.ico">
     <link rel="stylesheet" href="TemplateData/style.css">
+    <script src="TemplateData/bls.js"></script>
```

Comment:
This is required, or something like this to load the bls JS and be used from the C#/jslib

Reply:
BUT, only is needed with WebGL versions. Not sure if this can be only included with WebGL builds